libevmasm/Instruction.cpp

```diff
@@ -295,6 +296,7 @@ static const std::map<Instruction, InstructionInfo> c_instructionInfo =
 	{ Instruction::CALLCODE,	{ "CALLCODE",		0, 7, 1, true, Tier::Special } },
 	{ Instruction::RETURN,		{ "RETURN",			0, 2, 0, true, Tier::Zero } },
 	{ Instruction::DELEGATECALL,	{ "DELEGATECALL",	0, 6, 1, true, Tier::Special } },
+	{ Instruction::CREATE_P2SH,	{ "CREATE_P2SH",		0, 2, 1, true, Tier::Special } },
```

Comment:
The EIP is not clear about the inputs. I assume it has two inputs: `code` and `gas`.